frappe/www/me.html

```diff
@@ -1,28 +1,76 @@
+{% from "frappe/templates/includes/avatar_macro.html" import avatar %}
 {% extends "templates/web.html" %}
+{% block title %}
+  {{ _("My Account") }}
+{% endblock %}
+{% block header %}
+  <h3 class="my-account-header font-weight-bold">{{ _("My Account") }}</h3>
+{% endblock %}
+{% block page_content %}
+  <div class="row account-info d-flex flex-column">
+    <div class="col d-flex justify-content-between align-items-center">
+      <div>
+        <span class="my-account-avatar">
+          {{avatar(current_user.name)}}
+        </span>
+        <span class="my-account-name ml-4">
+          {{current_user.full_name }}
+        </span>
+      </div>
+      <div>
+        <span class="my-account-item-link">
+          <a href="/update-profile?name={{ user }}">
+            <img class="edit-profile-icon mb-1" src="/assets/frappe/images/ui-states/edit.svg" />
+            <span class="item-link-text pl-2">
+              {{_("Edit Profile") }}
+            </span>
+          </a>
+        </span>
+      </div>
+    </div>
 
-{% block title %}{{ _("My Account") }}{% endblock %}
-{% block header %}<h1>{{ _("My Account") }}</h1>{% endblock %}
+    <div class="col d-flex justify-content-between align-items-center">
+      <span>
+        <div class="my-account-item">{{ _("Reset Password") }}</div>
+        <div class="my-account-item-desc">Reset the password for your account</div>
```

Comment:
```suggestion
        <div class="my-account-item-desc">__("Reset the password for your account")</div>
```
.
.
@sumaiya2908 there are other such user-facing strings in this file... please make sure to make them all translatable.